data/pirate jobs.txt

```diff
@@ -1372,7 +1372,7 @@ mission "Slave Transport [0]"
 	job
 	repeat
 	description "These pirates need <bunks> slaves moved to <destination> along with <cargo>. Payment is <payment>. Slavery is highly illegal throughout the galaxy, so this may be a dangerous job to take."
-	passengers 2 10 .5
+	passengers 2 10 .9
 	cargo "shackles and grub" 1
 		illegal -1
```

Comment:
isn't there an atrocity tag now instead of setting illegal to -1? I think you'd need to replace all these with the atrocity tag